### PR TITLE
Feature request tracing

### PR DESCRIPTION
**Pull request by manu-khullar**

### CODE REVIEW

File: src/main/java/com/coronacarecard/trace/AddResponseHeaderFilter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response,
+                         FilterChain chain) throws IOException, ServletException {
+        HttpServletResponse httpServletResponse = (HttpServletResponse) response;
+        httpServletResponse.setHeader("tracer-id", tracer.currentSpan().context().traceIdString());
+        chain.doFilter(request, response);
+    }
+
+    @Override
```
Comment (debajyotimahanta) [inline src/main/java/com/coronacarecard/trace/AddResponseHeaderFilter.java:34]: these can be empty?

File: src/main/resources/bootstrap.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+spring:
```
Comment (debajyotimahanta) [inline src/main/resources/bootstrap.yml]: we can set it in appliation.properties as `spring.application.name=bar`

File: src/main/java/com/coronacarecard/trace/AddResponseHeaderFilter.java (hunk 1 above)
Comment (manu-khullar) [reply src/main/java/com/coronacarecard/trace/AddResponseHeaderFilter.java:34]: We are not overriding init and destroy methods. Init is used for using FilterConfig for any special initiallization. And we are using default destroy no special need to override destroy as well. Below are some details about them.


void init(FilterConfig filterConfig)
The servlet container calls init() as a filter is first instantiated and placed into service. This method takes a javax.servlet.FilterConfig instance as input, which the servlet container uses to pass information to the filter during the initialization. Include any special initialization requirements in your implementation. Also see "Methods of the FilterConfig Interface".


destroy(): 
The servlet container calls destroy() after all execution of the filter has completed (all threads of the doFilter() method have completed, or a timeout has occurred) and the filter is being taken out of service. Include any special cleanup requirements in your implementation.

File: src/main/resources/bootstrap.yml (hunk 2 above)
Comment (manu-khullar) [reply src/main/resources/bootstrap.yml]: I followed spring sleuth documentation so added that. Sure will make this change.

**Review by debajyotimahanta — APPROVED** [review]

**Pull request merged**